twitter/tests/test.bal

```diff
@@ -33,120 +38,283 @@ TwitterConfiguration twitterConfig = {
     accessTokenSecret: testAccessTokenSecret
 };
 
-Client twitterClient = new(twitterConfig);
+MediaParams mediaParams = {
+    media: "https://twitter.com/andypiper/status/903615884664725505",
+    //media:mediaIds,
+    possiblySensitive: false
+};
+
+LocationParams locationParams = {
+    placeInfo: (37.7821120598956, 122.400612831116),
+    //placeInfo:"df51dec6f4ee2b2c",
```

Comment:
Unnecessary line?

Reply:
Here we have two ways to pass placeInfo as (logtitude, latitude) or placeId.
Alternatively we can pass placeId. So, I'll add comment there.

https://github.com/wso2-ballerina/module-twitter/pull/83/files#diff-f6293dd6ea86eda702af7d079e754c83R75